pkg/image/registry/imagestreamimport/rest.go

```diff
@@ -104,33 +106,37 @@ func (r *REST) Create(ctx apirequest.Context, obj runtime.Object, _ bool) (runti
 	if !ok {
 		return nil, kapierrors.NewBadRequest("unable to get user from context")
 	}
-	isCreateImage, err := r.sarClient.Create(authorizationapi.AddUserToSAR(user,
-		&authorizationapi.SubjectAccessReview{
-			Action: authorizationapi.Action{
-				Verb:     "create",
-				Group:    imageapi.GroupName,
-				Resource: "images",
+	createImageSAR := authorizationutil.AddUserToSAR(user, &authorizationapi.SubjectAccessReview{
+		Spec: authorizationapi.SubjectAccessReviewSpec{
+			ResourceAttributes: &authorizationapi.ResourceAttributes{
+				Namespace: inputMeta.Namespace,
+				Verb:      "create",
+				Group:     imageapi.GroupName,
+				Resource:  "images",
 			},
 		},
-	))
+	})
+	isCreateImage, err := r.sarClient.Create(createImageSAR)
 	if err != nil {
 		return nil, err
 	}
 
-	isCreateImageStreamMapping, err := r.sarClient.Create(authorizationapi.AddUserToSAR(user,
-		&authorizationapi.SubjectAccessReview{
-			Action: authorizationapi.Action{
-				Verb:     "create",
-				Group:    imageapi.GroupName,
-				Resource: "imagestreammapping",
+	createImageStreamMappingSAR := authorizationutil.AddUserToSAR(user, &authorizationapi.SubjectAccessReview{
+		Spec: authorizationapi.SubjectAccessReviewSpec{
+			ResourceAttributes: &authorizationapi.ResourceAttributes{
+				Namespace: inputMeta.Namespace,
```

Comment:
I expected this one to be namespace scoped, but it looks like it wasn't namespaced before.  Was it?

Reply:
It was not, I'm removing the namespace parameter.